src/util/electron_helper.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <napi.h>
+#include <string>
+
+namespace zmq {
+bool hasRun = false;
+bool hasElectronMemoryCageCache = false;
```

Comment:
I think using the `static` keyword in the call site should be enough. Extra caching is not probably needed.